specification/signalr/resource-manager/readme.nodejs.md

```diff
@@ -0,0 +1,15 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-signalr
+  package-version: 1.0.0
```

Comment:
Are we sticking with this approach for now? (I can definitely understand choosing expediency for the current moment.)

Reply:
Yeah. For now let's keep with this, and then when we have more time we can come up with a better solution.